### ISSUE TITLE
[Feature Request] Replicate 4chan stylesheet in Thread HTML

### ISSUE THREAD

**Issue by korewatori** [murrty/YChanEx#17]
This tool works on Wine on Arch Linux from what I've tested, and functions completely fine in case you were wondering, but I was wondering if you could add an option that downloads the thread's HTML while still keeping the design of 4chan's threads (through the CSS). A program that does exactly this is [GChan](https://github.com/Issung/GChan) but that doesn't seem to work on Wine for me.

Thanks

**Comment by murrty** [comment]
TL;DR Unless someone wants to create a small and condensed version of 4chans' stylesheet that I can implement (with multi-file support), I'm not going to change from the custom HTML I made for this program. If you want more detailed reasoning, keep reading.

GChan did what was done before, saving the HTML and replacing portions of it. It does stop short of a problem that has been experienced through this application, mainly stylesheet changes. Even the original branch of this application (it is a fork from [ychan on sourceforge](https://sourceforge.net/projects/ychan/), but this is completely changed) had this same option with the same effect, but it broke some years ago because of stylesheet changes. A bunch of different ideas went through my head for this specific option,

* Download CSS per thread, but then you start saving redundant files for n-amount of threads
* Static CSS, but then you don't have version controlling without being on-time with 4chan
* Static CSS but with version controlling that version minor, inconsequential changes that cause insane amounts of version generations

and in the end I figured it'd be better to just make a small designed for all threads, and all chans (minus the HTML-parsed ones).

I'm not a HTML/CSS guy, so the result I did end up making is pretty simplistic, and it's not the prettiest, but it does its job without being a massive problem.

Again, if someone wants to make a base HTML+CSS based on 4chan (with multi-file support) then I can replace the one I have now with that, but as it stands for now, it's just gonna be as-is.
